portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,27 +27,40 @@
 @WebServlet("/comment")
 public class DataServlet extends HttpServlet {
 
+  private final String HTML_Contant_Type = "text/html";
+
+  private final String REDIRECT = "/index.html";
+
+  private final String ENTITY_TYPE = "comment";
+
+  private final String FIELD_NAME = "name";
+
+  private final String FIELD_TEXT = "text";
+  
+  private final String DEFAULT_VAL = "";
```

Comment:
Too much whitespace :) Related fields can be grouped together.

See an example here:
https://screenshot.googleplex.com/4BnMVgYvOqq